controller/TestPreviewer.php

```diff
@@ -111,4 +115,20 @@ private function getTestPreviewerPresetsMapService(): TestCategoryPresetMap
     {
         return $this->getPsrContainer()->get(TestCategoryPresetMap::class);
     }
+
+    private function mapXmlExceptionMessage(XmlStorageException $exception): string
+    {
+        if (
+            stristr(
```

Comment:
nitpick:
```suggestion
            strpos(
```

Reply:
Thanks, it works too but will stick with my version :)